.github/workflows/main_push.yml

```diff
@@ -1,17 +1,11 @@
 name: CI
 
+permissions: write-all
+
 on:
   push:
     branches:
       - develop
-  pull_request:
-    branches:
-      - develop
-    types:
-      - opened
-      - reopened
-      - synchronize
-      - labeled
 
   # Allows to run this workflow manually from the Actions tab
   workflow_dispatch:
```

Comment:
The concurrency group can be simplified to ` ${{ github.workflow }}-${{ github.ref_name }}`

Reply:
not even the fallback on `github.run-id`?

Reply:
https://github.com/Activiti/Activiti/pull/4529/commits/d5f96261875a6604378c59939a4d890b91061612